diff -ur logjam-4.4.1-orig/protocol/liblivejournal/livejournal/entry.c logjam-4.4.1/protocol/liblivejournal/livejournal/entry.c
--- logjam-4.4.1-orig/protocol/liblivejournal/livejournal/entry.c	2005-01-03 01:18:06.000000000 -0700
+++ logjam-4.4.1/protocol/liblivejournal/livejournal/entry.c	2005-05-02 21:53:56.000000000 -0600
@@ -43,6 +43,8 @@
 	memcpy(newe, e, sizeof(LJEntry));
 	if (e->subject)
 		newe->subject = g_strdup(e->subject);
+	if (e->url)
+		newe->url = g_strdup(e->url);
 	if (e->event)
 		newe->event = g_strdup(e->event);
 	if (e->mood)
@@ -56,6 +58,7 @@
 void
 lj_entry_free(LJEntry *e) {
 	g_free(e->subject);
+	g_free(e->url);
 	g_free(e->event);
 	g_free(e->mood);
 	g_free(e->music);
@@ -240,6 +243,7 @@
 		xmlSetProp(root, "itemid", buf);
 	}
 	XML_ENTRY_META_SET(subject);
+	XML_ENTRY_META_SET(url);
 	XML_ENTRY_META_SET(event);
 	if (entry->mood || entry->moodid) {
 		if (entry->mood)
@@ -361,6 +365,7 @@
 	verify_utf8(&entry->event);
 	entry->subject = g_strdup(lj_result_getf(result, "events_%d_subject", i));
 	verify_utf8(&entry->subject);
+	entry->url = g_strdup(lj_result_getf(result, "events_%d_url", i));
 	lj_security_from_strings(&entry->security, 
 			lj_result_getf(result, "events_%d_security", i),
 			lj_result_getf(result, "events_%d_allowmask", i));
@@ -586,6 +591,7 @@
 	RFC822_GET(subject)
 	else RFC822_GET(mood) /* XXX id */
 	else RFC822_GET(music)
+	else RFC822_GET(url)
 	else RFC822_GET(pickeyword)
 	else if (g_ascii_strcasecmp(key, "time") == 0) {
 		if (entry) lj_ljdate_to_tm(val, &entry->time);
@@ -720,6 +726,7 @@
 	
 	/* XXX: add all other metadata fields */
 	append_field(str, "Subject", entry->subject, includeempty);
+	append_field(str, "URL", entry->url, includeempty);
 	append_field(str, "Mood", entry->mood, includeempty);
 	append_field(str, "Music", entry->music, includeempty);
 	append_field(str, "PicKeyword", entry->pickeyword, includeempty);
diff -ur logjam-4.4.1-orig/protocol/liblivejournal/livejournal/entry.h logjam-4.4.1/protocol/liblivejournal/livejournal/entry.h
--- logjam-4.4.1-orig/protocol/liblivejournal/livejournal/entry.h	2005-01-03 01:18:06.000000000 -0700
+++ logjam-4.4.1/protocol/liblivejournal/livejournal/entry.h	2005-05-02 21:36:39.000000000 -0600
@@ -50,6 +50,7 @@
 	int itemid, anum;
 	char *subject;
 	char *event;
+	char *url;
 
 	int moodid;  /* a moodid is only meaningful in the context of a particular LJServer. */
 	char *mood, *music, *pickeyword;
