diff -ur logjam-4.5.1-o/configure logjam-4.5.1/configure
--- logjam-4.5.1-o/configure	2005-08-24 21:37:42.000000000 -0600
+++ logjam-4.5.1/configure	2006-04-08 03:21:29.000000000 -0600
@@ -19664,7 +19664,7 @@
 #define GETTEXT_PACKAGE "$GETTEXT_PACKAGE"
 _ACEOF
 
-ALL_LINGUAS="es en_GB en_US.UTF-8 de he ja ru_RU uk_UA"
+ALL_LINGUAS="es en_GB en_US de he ja ru uk"
 
 
 for ac_header in locale.h
Only in logjam-4.5.1: configure~
diff -ur logjam-4.5.1-o/configure.in logjam-4.5.1/configure.in
--- logjam-4.5.1-o/configure.in	2005-08-24 21:36:59.000000000 -0600
+++ logjam-4.5.1/configure.in	2006-04-08 03:21:06.000000000 -0600
@@ -18,7 +18,7 @@
 GETTEXT_PACKAGE=logjam
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE],"$GETTEXT_PACKAGE",[gettext package])
-ALL_LINGUAS="es en_GB en_US.UTF-8 de he ja ru_RU uk_UA"
+ALL_LINGUAS="es en_GB en_US de he ja ru uk"
 AM_GLIB_GNU_GETTEXT
 
 dnl Even though the Unix versions don't use gthread, the soup
Only in logjam-4.5.1: configure.in~
Only in logjam-4.5.1: configure.in.orig
Only in logjam-4.5.1: configure.orig
